ui/perfherder/tests/TestsView.jsx

```diff
@@ -113,7 +113,7 @@ class HeathView extends React.PureComponent {
     const frameworkNames =
       frameworks && frameworks.length ? frameworks.map(item => item.name) : [];
 
-    const healthDropdowns = [
+    const testsDropdowns = [
```

Comment:
This is a local variable. I think it's pretty clear what the namespace is.
How about renaming this to just "dropdowns"?